blockchain/types/tx_internal_data_chain_data_anchoring.go

```diff
@@ -280,18 +281,28 @@ func (t *TxInternalDataChainDataAnchoring) Execute(sender ContractRef, vm VM, st
 }
 
 func (t *TxInternalDataChainDataAnchoring) MakeRPCOutput() map[string]interface{} {
+	decoded, err := DecodeAnchoringDataToJSON(t.Payload)
+	if err != nil {
+		logger.Error("decode anchor payload", "err", err)
```

Comment:
The error messages are the same as below fee-delegated ones. Isn't it necessary to distinguish them?